config/config_test.go

```diff
@@ -1,62 +0,0 @@
-package config
```

Comment:
I think I remember you saying that you were going to rebase all this stuff later, but yeah this change is out of context for the commit.